# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/hsetroot/hotfix-inst-wrapper.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- hsetroot-1.0.5/Makefile.vanilla	2025-04-10 22:40:20.003593819 +0200
+++ hsetroot-1.0.5/Makefile	2025-04-10 22:40:04.323592244 +0200
@@ -30,8 +30,8 @@
 hsr-outputs: hsr-outputs.o
 
 install: hsetroot hsr-outputs
-	install -st $(PREFIX)/bin/ hsetroot
-	install -st $(PREFIX)/bin/ hsr-outputs
+	install hsetroot $(PREFIX)/bin/
+	install hsr-outputs $(PREFIX)/bin/
 
 clean:
 	rm -f *.o hsetroot hsr-outputs
